internal/clients/nightfall/nightfall.go

```diff
@@ -303,26 +365,26 @@ func (n *Client) Scan(ctx context.Context, items []string) (*nf.ScanTextResponse
 // contains sensitive data
 func (n *Client) ReviewDiff(ctx context.Context, logger logger.Logger, fileDiffs []*diffreviewer.FileDiff) ([]*diffreviewer.Comment, error) {
 	fileDiffs = filterFileDiffs(fileDiffs, n.FileInclusionList, n.FileExclusionList, logger)
-	contentToScanList := make([]*contentToScan, 0, len(fileDiffs))
-	// Chunk fileDiffs content and store chunk and its metadata
+	//contentToScanList := make([]*contentToScan, 0, len(fileDiffs))
+	fileToScanList := make([]*fileToScan, 0, len(fileDiffs))
+
 	for _, fd := range fileDiffs {
-		for _, hunk := range fd.Hunks {
-			for _, line := range hunk.Lines {
-				chunkedContent, err := chunkContent(contentChunkByteSize, line, fd.PathNew)
-				if err != nil {
-					logger.Error("Error chunking git diff")
-					return nil, err
-				}
-				contentToScanList = append(contentToScanList, chunkedContent...)
-			}
+		file, err := getFileToScan(fd)
+		if err != nil {
+			return nil, err
 		}
+		if len(file.Content) > maxAPIRequestSize {
```

Comment:
this seems like a new constraint that did not exist before these changes. Should we just chunk the file?